cfme/tests/configure/test_paginator.py

```diff
@@ -101,6 +102,13 @@ def schedule(appliance):
 @pytest.mark.parametrize('place_info', general_list_pages,
                          ids=['{}_{}'.format(set_type[0], set_type[2].lower())
                               for set_type in general_list_pages])
+@pytest.mark.meta(blockers=[BZ(1724747, forced_streams=['5.11'], unblock=lambda place_info:
```

Comment:
Another annoying formatting remark, but to increase readability, I'd do something like
```python,
@pytest.mark.meta(blockers=[
    BZ(1724747, forced_streams=['5.11'], unblock=lambda place_info: '{}_{}'.format(
        place_info[0], place_info[2].lower()
    ) not in BAD_ZONES),
    BZ(1726345, forced_streams=['5.11'], unblock=lambda place_info: '{}_{}'.format(
        place_info[0], place_info[2].lower()
    ) not in ['regions_serversbyroles'])
])
```
Where `BAD_ZONES = ['zones_diagnostics', 'zones_serversbyroles', 'zones_servers', 'zones_candugapcollection', 'zones_rolesbyservers', 'zones_collectlogs']`

Reply:
Leaving this as _optional_